tests/test_meta/test_decay_estimator.py

```diff
@@ -58,3 +55,20 @@ def test_throw_warning(mod):
         DecayEstimator(mod, decay=0.95).fit(X, y)
         assert "sample_weight" in str(e)
         assert type(mod).__name__ in str(e)
+
+
+@pytest.mark.parametrize(
```

Comment:
In principle there are _three_ functionalities for checking estimator type (`is_regressor`, `is_classifier`, `is_outlier_detector`). The shortcut in here is to use a boolean flag to check regressor or classifier only.